scss/intro.scss

```diff
@@ -9,6 +9,14 @@
 .intro__img {
 	flex: 0 100%;
 	width: 50%;
+
+	&.rounded {
+		border-radius: 3.5rem;
```

Comment:
How about using a percent here as well so that it looks similar no matter the image's size (for example, on a phone).

Reply:
This has been changed from rem notation to percentage.